README.md

```diff
@@ -18,23 +20,34 @@
 3. Install the requirements (one time instruction):
 `pip install -r requirements.txt`
 
-4. If you run the server locally, set up the database for local development
+4. Set up the database via docker or connect your own Postgres instance
 `docker-compose up db`
 
-5. Run the server from the root folder:
+5. Run the migrations from the root folder:
+`python src/manage.py migrate`
+
+6. Run the server from the root folder:
 `python src/manage.py runserver` The server runs on http://127.0.0.1:8000/
 
-6. Run the tests:
+7. To run the tests:
 `python src/manage.py test`
 
-7. Run pylint:
+8. To run pylint:
 `find src -name "*.py" | xargs pylint`
 
-8. Simulate a deployment locally with Django debug mode enabled (runs on port 8000):
+# Setup (Run via docker)
+
+This method should be sufficient for most development tasks as any (file) changes you make are monitored by StatReloader. To start the environment, run:
 `docker-compose up --build`
 
 Note: remove `--build` to skip building the container, will use the cached one (last build)
 
+After this, you can access the application on http://127.0.0.1:8000/ and a pgAdmin instance on http://127.0.0.1:5050/ with `admin@admin.com:admin` for username:password. From pgAdmin add a new server by giving it a name and the following database credentials:
+
+![image](https://user-images.githubusercontent.com/15870306/145728488-ada8aacf-ec53-42d1-8e4d-b7198c70cc77.png)
+
+
+
 # Deployment
 
 1. Build and run Docker deployment script (runs on port 80 - default web port):
```

Comment:
Not this line but line 2 and 3 here (Github doesn't allow comments on lines too far out of the changeset):

It should not be necessary to manually run migrations, if this is the case then we should open an issue to investigate this